packages/jest-leak-detector/src/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Before CI tells you :D please run `yarn jest-leak-detector -u` to update stale snapshots (you renamed the file and a new snapshot was added but old one not removed)

Reply:
Omg … :D